optima/optimization.py

```diff
@@ -58,25 +61,36 @@ def getresults(self):
 
 
 
-def defaultobjectives(which='outcome', verbose=2):
+def defaultobjectives(project=None, progset=None, which='outcome', verbose=2):
     """
     Define default objectives for the optimization. Some objectives are shared
     between outcome and money minimizations, while others are different. However,
     outcome minimization is performed as part of money minimization, so it's useful
     to keep all the keys for both. Still, ugly.
     
-    Version: 2016jan26
+    Version: 2016feb03
     """
 
     printv('Defining default objectives...', 3, verbose=verbose)
+    
+    if type(progset)==Programset:
+        defaultbudget = sum(progset.getdefaultbudget()[:])
+    if type(project)==Programset: # Not actually a project, but proceed anyway
+        defaultbudget = sum(project.getdefaultbudget()[:])
+    elif project is not None:
+        if progset is None: progset = 0
```

Comment:
i think it's good to print a warning whenever this is done, but not urgent for now